sdk/typescript/src/client/client.ts

```diff
@@ -551,14 +553,46 @@ export class IotaClient {
     /**
      * Return the latest IOTA system state object on networks supporting protocol version `>= 5`.
      * These are networks with node software release version `>= 0.11`.
+     *
+     * You probably want to use `getSupportedIotaSystemState` instead.
```

Comment:
I would say "you probably want to use" but rather explain the other exists and why is it useful